llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -516,10 +522,7 @@ class FilterChooser {
   // This returns a lit of undecoded bits of an instructions, for example,
```

Comment:
nit: list of instead if lit of? I know this PR is not touching it, but can you please fix the typo?